at_secondary/at_persistence_secondary_server/lib/src/keystore/hive_keystore_helper.dart

```diff
@@ -41,10 +42,11 @@ class HiveKeyStoreHelper {
             isEncrypted: isEncrypted,
             dataSignature: dataSignature,
             sharedKeyEncrypted: sharedKeyEncrypted,
-            publicKeyChecksum: publicKeyChecksum)
+            publicKeyChecksum: publicKeyChecksum,
+            encoding: encoding)
         .build();
-    at_data.metaData!.version = 0;
-    return at_data;
+    atData.metaData!.version = 0;
```

Comment:
Check with @murali-shris what the intent of metadata version is - should we be incrementing the metadata version because of this change, and what effects would that have elsewhere?